model_compression_toolkit/qunatizers_infrastructure/keras/quantize_wrapper.py

```diff
@@ -172,11 +179,28 @@ def call(self, inputs, training=None, **kwargs):
 
             self.set_quantize_weights(quantized_weights)
 
-            args = tf_inspect.getfullargspec(self.layer.call).args
-            if TRAINING in args:
+            args_spec = tf_inspect.getfullargspec(self.layer.call).args
+            if TRAINING in args_spec:
                 outputs = self.layer.call(inputs, training=training, **kwargs)
             else:
                 outputs = self.layer.call(inputs, **kwargs)
+            if self.dispatcher.num_act_quantizers > 0:
```

Comment:
maybe add space and comment "Quantize all activations" so it is clear that this block deals with activations